classification/notebooks/24_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -228,326 +197,323 @@
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/45.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/41.jpg, 18 files out of an estimated total of 138\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/can/23.jpg, 19 files out of an estimated total of 138\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/can/1.jpg, 20 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/46.jpg\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/47.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/can/12.jpg, 21 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/48.jpg\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/49.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/can/16.jpg, 22 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/50.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/can/31.jpg, 23 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/51.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/can/26.jpg, 24 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/52.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/47.jpg, 25 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/53.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/44.jpg, 26 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/54.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/49.jpg, 27 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/55.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/53.jpg, 28 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/56.jpg\n",
-      "Uploaded /Users/richinjain/projects/ComputerVision/data/fridgeObjects/can/29.jpg, 29 files out of an estimated total of 138\n",
-      "Uploading /Users/richinjain/projects/ComputerVision/data/fridgeObjects/carton/57.jpg\n"
+      "Uploading an estimated of 137 files\n",
```

Comment:
I think we can hide the output from this. It takes up a lot of room

Reply:
Done.